e2e/meradm/meradm_test.go

```diff
@@ -82,6 +82,19 @@ var _ = Describe("Meradm", func() {
 			_, err = meradmErrored("service", "add", "service1")
 			Expect(err).To(HaveOccurred(), "should have failed validation")
 		})
+
+		It("can set a healthcheck", func() {
+			meradm("service", "add", "service1", "tcp", "10.1.1.1:888", "-s=wrr", "-b=flag-1,flag-2")
+			meradm("service", "edit", "service1",
+				"--health-endpoint=http:556/health", "--health-period=5s", "--health-timeout=1s",
```

Comment:
This is wrong, the endpoint needs to be a valid url eg `http://:556/health`. Need validation in the server for this, probably.